packages/server/src/connector/index.ts

```diff
@@ -188,6 +197,12 @@ async function subscribeEvents() {
       }
 
       if (event.method === 'Slash') {
+        if (
+          event.data[0].toString().toLowerCase() ===
```

Comment:
Are we sure the validator stash address is case insensitive?

Reply:
Good point! They are not.